src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -97,6 +82,7 @@ typedef struct otPlatformConfig
  *       when initialization of OpenThread's drivers is most appropriate.
  *
  * @param[in]  aPlatformConfig  Platform configuration structure.
+ * @param[in]  aRadioUrl        Radio url.
```

Comment:
The below function `otSysInit` doesn't take this parameter. Does the function need this parameter?